process/throttle/antiflood/p2pAntiflood.go

```diff
@@ -183,6 +184,26 @@ func (af *p2pAntiflood) SetDebugger(debugger process.AntifloodDebugger) error {
 	return nil
 }
 
+// BlacklistPeer will add a peer to the black list
+func (af *p2pAntiflood) BlacklistPeer(peer core.PeerID, reason string, duration time.Duration) {
```

Comment:
in the future if we found enough reasons to make this worth, we can define a set of reasons and the blacklist duration associated.

Reply:
Agree, first let's see what we need